recipes/demquery/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set version = "0.2.0" %}
+{% set name = "demquery" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 19650a37a5da271af5697d429db8f6cf6d43804dc46113fa622b80acd982d1f8
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.5.0
+    - pip
```

you will need to add ``twine`` here
```suggestion
    - pip
    - twine
```